docs/tech-reference/rsync-distributor.rst

```diff
@@ -0,0 +1,105 @@
+=================
```

Comment:
This won't build by Sphinx. Can you fix all sphinx build errors by building the docs yourself.